src/data.js

```diff
@@ -1,117 +1,104 @@
+			"cat": {
+				text: 'Cat',
+				nextStep: 6
+			},
+			"pitbull": {
+				text: "Pit Bull",
+				nextStep: 6
+			},
+			"other": {
+				text: 'Another Type of Dog',
+				nextStep: 8
+			}
+		}
+	},
+	{
         id: 6,
-        text: "Sorry I get really excited. Which type of ${favoriteColor} do you prefer?",
-        sender: "bot",
-        key: "favoriteShade",
+        text: "Do you receive public assistance?",
+		sender: "question",
+		key: "receivesPublicAssistance",
         fieldType: "radio",
-        options: [
-            {
-                text: "Light",
-                value: "light",
-            },
-            {
-                text: "Regular",
-                value: "regular"
-            },
-            {
-                text: "Dark",
-                value: "dark"
-            }
-        ]
-    },
-    {
-        id: 7,
-        text: "Now that I know you even more, I understand you want to report a problem. Please briefly describe your problem below and I will try to help you with it.",
+        options: {
+			"true": {
+				text: 'Yes',
+				nextStep: 11
+			},
+			"false": {
+				text: 'No',
+				nextStep: 8
+			}
+		}
+	},
+	{
+		id: 8,
+        text: "What is your zip code?",
         sender: "bot",
-        key: "serviceType",
+        key: "zipCode",
         fieldType: "text",
-        onSubmit: ["checkKeywords"]
-    },
-    {
-        id: 8,
-        text: "Great, I am glad to help you with your ${serviceType} Request.",
-        sender: "bot"
-    },
-    {
-        id: 9,
-        text: "An address is required to submit a service request, please type your address.",
-        sender: "bot",
-        fieldType: "text",
-        key: "addresss",
-        validationTypes: ['address'],
-        onSubmit: ["googleMap", "checkForExistingRequest"]
-    },
-    {
+		validationTypes: ['zipCode'],
+		options: (answers) => {
+			//Return the next step
+			const zip = answers.zipCode ? parseInt(answers.zipCode) : null;
+			const dundalkCenterZips = [21222, 21219, 21220, 21221, 21224, 21237];
+			const zipIsInDundalkZips = dundalkCenterZips.indexOf(zip) > -1;
+
+			if (zipIsInDundalkZips) {
+				return 12;
+			}
+
+			const swapCenterZips = [21227, 21228, 21244, 21207];
+			const isCat = answers.petType ? answers.petType.toLowerCase().indexOf('cat') > -1 : false;
+			const zipIsInSwapZips = swapCenterZips.indexOf(zip) > -1;
+
+			if (isCat && zipIsInSwapZips) {
+				return 13;
+			}
+			return 14;
+		}
+	},
+    { // Non resident
         id: 10,
-        text: "Well it was nice getting to know you, here is what I found out. Your name is ${fullName} and really like ${favoriteShade} ${favoriteColor}. Have a great day.",
-        sender: "bot",
-        isLastMessage: true
-    },
-    {
+        text: "<div id=\"spayNeuterFormResults\" class=\"bc-form-control alert-warning\" aria-hidden=\"false\" role=\"alert\"><p>We're sorry. Only County residents are eligible for discount spay or neuter procedures.</p></div>",
```

Comment:
For this, and the other messages below, I think you should consider just passing through the message instead of the message and the HTML.

Reply:
hahahahha @mxsnyder did you look at any of these? fixing

Reply:
done